src/QueryList.php

```diff
@@ -99,6 +99,8 @@ public static function config()
     public function destruct()
     {
         phpQuery::$documents = [];
+        QueryList::$plugins=[];

```

Comment:
Since `QueryList::$plugins` is `static`, isn't this change wrong if there are multiple `QueryList` instances?